### PR TITLE
BP adding VERSION file, __version__ option and bump version

### PR DESCRIPTION
**Pull request by laq0110**

### CODE REVIEW

File: gcd/VERSION [hunk 1]
```diff
@@ -0,0 +1 @@
+1.1
```
Comment (danielfujii-jampp) [inline gcd/VERSION]: `1.1.0`

File: gcd/__init__.py [hunk 2]
```diff
@@ -9,4 +9,9 @@
         pyximport.install(reload_support=True)
         del pyimport
 
+with open(os.path.join(os.path.dirname(__file__), "VERSION"), 'r') as vf:
+    my_version = vf.read().strip()
+
+__version__ = my_version
```
Comment (danielfujii-jampp) [inline gcd/__init__.py]: Just curious, do you really need this?

File: CHANGELOG.rst [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+==========
+
+All notable changes to this project will be documented here.
+
+.. contents::
+
+v1.1.0 - 2019-09-24
+-------------------
+
+Added
```
Comment (danielfujii-jampp) [inline CHANGELOG.rst]: change to "Major" or sth similar. not "Added" IMO

Comment (laq0110) [reply CHANGELOG.rst]: Done!

File: gcd/__init__.py (hunk 2 above)
Comment (memeplex) [reply gcd/__init__.py]: Please leave two blank lines between top-level constructs. (PEP 8)

File: setup.py [hunk 4]
```diff
@@ -1,9 +1,12 @@
+import os
 from setuptools import setup
 
+with open(os.path.join(os.path.dirname(__file__), "gcd/VERSION"), 'r') as vf:
+    my_version = vf.read().strip()
 
 setup(
```
Comment (memeplex) [inline setup.py]: Please leave two blank lines between top-level constructs. (PEP 8)

**Pull request merged**